### PR TITLE
(port) Lizard Food: a Taste of the Homeworld (drinks and plants only)

### PR DESCRIPTION
**Pull request by boodaliboo**
original pr:https://github.com/tgstation/tgstation/pull/59412
New Drinks
The chef isn't the only one getting love here!
image

Some new bar drinks are also available, using two new lizardy drinks: Kortara, the product of fermenting Korta Nuts, and Mushroom Tea, made from Lavaland mushrooms (specifically the ones that give shavings). You can even turn your Mushroom Tea into delicious Mushi Kombucha!
Recipes can be seen here: https://pastebin.com/g2JXFcEA

Korta Nuts
image

Korta Nuts are a versatile tool in lizard cuisine, with two key ingredients coming from them: korta flour, made by grinding them, and korta milk, made by juicing them. They can also be mutated into Sweet Korta Nuts, which produce korta nectar: a sweet syrup used in a few dishes. They can be grown at botany, with the seeds being located in the Megaseed Vendor.
:cl:  
rscadd: Added three new botany plants korta nut, sweet korta nut, and sereka mushrooms, added 8 new reagents related to the new plants and stuff added

/:cl:

### CODE REVIEW

**Comment by Skrem7** [comment]
Fix the checks first then I'll go over flavor to make it lore-compatible.

Sprites look cool.

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -2394,3 +2394,115 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	taste_description = "regret"
+	glass_icon_state = "protein_blend"
+	glass_name = "Protein Blend"
+	glass_desc = "Vile, even by lizard standards."
+	nutriment_factor = 3 * REAGENTS_METABOLISM
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+/datum/reagent/consumable/ethanol/protein_blend/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
+	M.adjust_nutrition(2 * REM * delta_time)
+	if(!islizard(M))
```
Comment (redmoogle) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2465]: flip this

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 2]
```diff
@@ -2394,3 +2394,115 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	taste_description = "victory"
+	glass_icon_state = "triumphal_arch"
+	glass_name = "Triumphal Arch"
+	glass_desc = "A toast to the Empire, long may it stand."
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+/datum/reagent/consumable/ethanol/triumphal_arch/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
+	if(islizard(M))
+		SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "triumph", /datum/mood_event/memories_of_home, name)
+	..()
```
Comment (redmoogle) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]:
```suggestion
	return ..()
```

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 3]
```diff
@@ -1026,4 +1026,19 @@
...
+	taste_description = "mushrooms"
+	glass_icon_state = "mushroom_tea_glass"
+	glass_name = "glass of mushroom tea"
+	glass_desc = "Oddly savoury for a drink."
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+
+/datum/reagent/consumable/mushroom_tea/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
+	if(islizard(M))
+		M.adjustOxyLoss(-0.5 * REM * delta_time, 0)
+	..()
```
Comment (redmoogle) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm:1042]: What the hell is happening here

**Review by cuackles — APPROVED** [review]
sprites aproved

**Comment by Marmio64** [comment]
korta fit deeze nuts in yo mouf

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm (hunk 3 above)
Comment (boodaliboo) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm:1042]: what the fuck is delta time is my conundrum working on that

**Comment by boodaliboo** [comment]
this 

> korta fit deeze nuts in yo mouf

this is my excuse to be more racist to lizards

**Comment by JamieD1** [comment]
Compile

**Comment by boodaliboo** [comment]
need help figuring the last two merge errors

**Comment by boodaliboo** [comment]
what the fuck how is the code breaking in the vending machine section

File: code/modules/hydroponics/grown/korta_nut.dm [hunk 4]
```diff
@@ -0,0 +1,51 @@
+//Korta Nut
+/obj/item/seeds/korta_nut
+	name = "pack of korta nut seeds"
+	desc = "These seeds grow into korta nut bushes, native to Tizira."
```
Comment (Skrem7) [inline code/modules/hydroponics/grown/korta_nut.dm]:
```suggestion
	desc = "These seeds grow into korta nut bushes, native to Sangris."
```

File: code/modules/hydroponics/grown/korta_nut.dm [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+	icon_grow = "kortanut-grow"
+	icon_dead = "kortanut-dead"
+	genes = list(/datum/plant_gene/trait/repeated_harvest)
+	mutatelist = list(/obj/item/seeds/korta_nut/sweet)
+	reagents_add = list(/datum/reagent/consumable/nutriment/vitamin = 0.04, /datum/reagent/consumable/nutriment = 0.1)
+
+/obj/item/reagent_containers/food/snacks/grown/korta_nut
+	seed = /obj/item/seeds/korta_nut
+	name = "korta nut"
+	desc = "A little nut of great importance. Has a peppery shell which can be ground into flour and a soft, pulpy interior that produces a milky fluid when juiced. Or you can eat them whole, as a quick snack."
```
Comment (Skrem7) [inline code/modules/hydroponics/grown/korta_nut.dm]:
```suggestion
	desc = "A small nut. Has a peppery shell that can be ground into flour. Inside is a soft, pulpy interior that produces a milky fluid when juiced. Or you can eat them whole, as a quick snack."
```

File: code/modules/hydroponics/grown/korta_nut.dm [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+	icon_state = "korta_nut"
+	grind_results = list(/datum/reagent/consumable/korta_flour = 0)
+	juice_results = list(/datum/reagent/consumable/korta_milk = 0)
+	tastes = list("peppery heat" = 1)
+	distill_reagent = /datum/reagent/consumable/ethanol/bilk/kortara
+
+//Sweet Korta Nut
+/obj/item/seeds/korta_nut/sweet
+	name = "pack of sweet korta nut seeds"
+	desc = "These seeds grow into sweet korta nuts, a mutation of the original species that produces a thick syrup that Tizirans use for desserts."
```
Comment (Skrem7) [inline code/modules/hydroponics/grown/korta_nut.dm]:
```suggestion
	desc = "These seeds grow into sweet korta nuts, a mutation of the original species that produces a thick syrup that vuulen use in desserts."
```

File: code/modules/mining/lavaland/ash_flora.dm [hunk 7]
```diff
@@ -324,3 +325,36 @@
 				A.initialize_custom_food(src, S, user)
 	else
 		. = ..()
+
+/obj/structure/flora/ash/seraka
+	icon_state = "seraka_mushroom"
+	name = "seraka mushrooms"
+	desc = "A small cluster of seraka mushrooms. These must have come with the ashlizards."
```
Comment (Skrem7) [inline code/modules/mining/lavaland/ash_flora.dm]:
```suggestion
	desc = "A small cluster of seraka mushrooms, normally found on Sangris. Strange that they're here..."
```

File: code/modules/mining/lavaland/ash_flora.dm [hunk 8]
```diff
@@ -324,3 +325,36 @@
...
+	needs_sharp_harvest = FALSE
+	harvested_name = "harvested seraka mushrooms"
+	harvested_desc = "A couple of small seraka mushrooms, with the larger ones clearly having been recently removed. They'll grow back... eventually."
```
Comment (Skrem7) [inline code/modules/mining/lavaland/ash_flora.dm]:
```suggestion
	harvested_desc = "What remains of a harvested patch of seraka mushrooms. They are capable of regrowing quickly."
```

File: code/modules/mining/lavaland/ash_flora.dm [hunk 9]
```diff
@@ -324,3 +325,36 @@
...
+	harvest = /obj/item/reagent_containers/food/snacks/grown/ash_flora/seraka
+	harvest_amount_high = 6
+	harvest_time = 25
+	harvest_message_low = "You pluck a few choice tasty mushrooms."
+	harvest_message_med = "You grab a good haul of mushrooms."
+	harvest_message_high = "You hit the mushroom motherlode and make off with a bunch of tasty mushrooms."
```
Comment (Skrem7) [inline code/modules/mining/lavaland/ash_flora.dm:341]:
```suggestion
	harvest_message_high = "You hit the mushroom motherlode and harvest several from the batch."
```

File: code/modules/mining/lavaland/ash_flora.dm [hunk 10]
```diff
@@ -324,3 +325,36 @@
...
+	regrowth_time_low = 3000
+	regrowth_time_high = 5400
+
+/obj/item/reagent_containers/food/snacks/grown/ash_flora/seraka
+	name = "seraka cap"
+	desc = "Small, deeply flavourful mushrooms originally native to Tizira."
```
Comment (Skrem7) [inline code/modules/mining/lavaland/ash_flora.dm:347]:
```suggestion
	desc = "Small, deeply flavourful mushrooms originally native to Sangris. These ones are flaked in ash and soot."
```

File: code/modules/mining/lavaland/ash_flora.dm [hunk 11]
```diff
@@ -324,3 +325,36 @@
...
+	icon_state = "seraka_cap"
+	seed = /obj/item/seeds/lavaland/seraka
+
+/obj/item/seeds/lavaland/seraka
+	name = "pack of seraka mycelium"
+	desc = "This mycelium grows into seraka mushrooms, a species of savoury mushrooms originally native to Tizira used in food and traditional medicine."
```
Comment (Skrem7) [inline code/modules/mining/lavaland/ash_flora.dm:353]:
```suggestion
	desc = "This mycelium grows into seraka mushrooms, a species of savoury mushrooms originally native to Sangris. They are most commonly used in food and traditional medicine."
```

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 12]
```diff
@@ -2394,3 +2394,104 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_name = "Glass of Amber Moon"
 	glass_desc = "A strange cocktail with a cracked egg believed to treat hangovers."
 	shot_glass_icon_state = "ambermoonshotglass"
+
+/datum/reagent/consumable/ethanol/bilk/kortara
+	name = "Kortara"
+	description = "A sweet, milky nut-based drink enjoyed on Tizira. Frequently mixed with fruit juices and cocoa for extra refreshment."
```
Comment (Skrem7) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2400]:
```suggestion
	description = "A sweet, milky nut-based drink traditional in vuulek cuisine. Frequently mixed with fruit juices and cocoa for extra refreshment."
```

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 13]
```diff
@@ -2394,3 +2394,104 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	quality = DRINK_GOOD
+	taste_description = "sweet nectar"
+	glass_icon_state = "kortara_glass"
+	glass_name = "glass of kortara"
+	glass_desc = "The fermented nectar of the Korta nut, as enjoyed by lizards galaxywide."
+
+	
+/datum/reagent/consumable/ethanol/sea_breeze
+	name = "Sea Breeze"
+	description = "Light and refreshing with a mint and cocoa hit- like mint choc chip ice cream you can drink!"
```
Comment (Skrem7) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2412]:
```suggestion
	description = "Light and refreshing with a hint of mint and cocoa. Sweet, like a smoothie."
```

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 14]
```diff
@@ -2394,3 +2394,104 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	boozepwr = 15
+	color = "#CFFFE5"
+	quality = DRINK_VERYGOOD
+	taste_description = "mint choc chip"
+	glass_icon_state = "sea_breeze"
+	glass_name = "Sea Breeze"
+	glass_desc = "Minty, chocolatey, and creamy. It's like drinkable mint chocolate chip!"
```
Comment (Skrem7) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2419]:
```suggestion
	glass_desc = "A mint-chocolate, creamy shake."
```

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 15]
```diff
@@ -2394,3 +2394,104 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+
+/datum/reagent/consumable/ethanol/sea_breeze/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
+	M.apply_status_effect(/datum/status_effect/throat_soothed)
+	..()
+
+/datum/reagent/consumable/ethanol/white_tiziran
+	name = "White Tiziran"
+	description = "A mix of vodka and kortara. The Lizard imbibes."
```
Comment (Skrem7) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2427]:
```suggestion
	description = "A mix of vodka and kortara, often utilized during vuulek celebrations."
```

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 16]
```diff
@@ -2394,3 +2394,104 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	boozepwr = 65
+	color = "#A68340"
+	quality = DRINK_GOOD
+	taste_description = "strikes and gutters"
+	glass_icon_state = "white_tiziran"
+	glass_name = "White Tiziran"
+	glass_desc = "I had a rough night and I hate the fucking humans, man."
```
Comment (Skrem7) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2434]:
```suggestion
	glass_desc = "A sweet mint vodka with a hint of cocoa."
```

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 17]
```diff
@@ -2394,3 +2394,104 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+
+/datum/reagent/consumable/ethanol/drunken_espatier
+	name = "Drunken Espatier"
+	description = "Look, if you had to get into a shootout in the cold vacuum of space, you'd want to be drunk too."
```
Comment (Skrem7) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2438]:
```suggestion
	description = "A drink concocted by vuulek warriors for traditional duels. Strong and numbing."
```

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 18]
```diff
@@ -2394,3 +2394,104 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	boozepwr = 65
+	color = "#A68340"
+	quality = DRINK_GOOD
+	taste_description = "sorrow"
+	glass_icon_state = "drunken_espatier"
+	glass_name = "Drunken Espatier"
+	glass_desc = "A drink to make facing death easier."
```
Comment (Skrem7) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2445]:
```suggestion
	glass_desc = "A drink that numbs the body, making it difficult to be aware of injury."
```

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 19]
```diff
@@ -2394,3 +2394,104 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+/datum/reagent/consumable/ethanol/protein_blend
+	name = "Protein Blend"
+	description = "A vile blend of protein, pure grain alcohol, korta flour, and blood. Useful for bulking up, if you can keep it down."
+	boozepwr = 65
+	color = "#FF5B69"
+	quality = DRINK_NICE
+	taste_description = "regret"
+	glass_icon_state = "protein_blend"
+	glass_name = "Protein Blend"
+	glass_desc = "Vile, even by lizard standards."
```
Comment (Skrem7) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2461]:
```suggestion
	glass_desc = "A vile-looking drink that's hard to stomach, even if nutritious."
```

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 20]
```diff
@@ -2394,3 +2394,104 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	if(!islizard(M))
+		M.adjust_nutrition(2 * REM * delta_time)
+		M.adjust_disgust(5 * REM * delta_time)
+	else
+		M.adjust_disgust(2 * REM * delta_time)
+	..()
+
+/datum/reagent/consumable/ethanol/mushi_kombucha
+	name = "Mushi Kombucha"
+	description = "A popular summer beverage on Tizira, made from sweetened mushroom tea."
```
Comment (Skrem7) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:2474]:
```suggestion
	description = "A popular mushroom tea among vuulen, traditionally enjoyed during blistering days of heat."
```

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 21]
```diff
@@ -2394,3 +2394,104 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	color = "#C46400"
+	quality = DRINK_VERYGOOD
+	taste_description = "sweet 'shrooms"
+	glass_icon_state = "glass_orange"
+	glass_name = "glass of mushi kombucha"
+	glass_desc = "A glass of (slightly alcoholic) fermented sweetened mushroom tea. Refreshing, if a little strange."
+
+/datum/reagent/consumable/ethanol/triumphal_arch
+	name = "Triumphal Arch"
+	description = "A drink celebrating the Lizard Empire and its military victories. It's popular at bars on Unification Day."
```
Comment (Skrem7) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]:
```suggestion
	description = "A drink celebrating the Opsillian Republic and its rapid growth. A popular tool of integration efforts."
```

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 22]
```diff
@@ -2394,3 +2394,104 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	boozepwr = 60
+	color = "#FFD700"
+	quality = DRINK_FANTASTIC
+	taste_description = "victory"
+	glass_icon_state = "triumphal_arch"
+	glass_name = "Triumphal Arch"
+	glass_desc = "A toast to the Empire, long may it stand."
```
Comment (Skrem7) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]:
```suggestion
	glass_desc = "A toast to Sangris, the jewel of the vuulen."
```

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 23]
```diff
@@ -1026,4 +1026,18 @@
 		to_chat(M, "<span class = 'notice'>[pick("Buzz Buzz.", "Stinging with flavour.", "Ideal of the worker drone", "A Hive of Flavour", "Sap back that missing energy!", "Got Honey?", "The Queen approved it!")]</span>")
 	..()
 
-
+/datum/reagent/consumable/mushroom_tea
+	name = "Mushroom Tea"
+	description = "A savoury glass of tea made from polypore mushroom shavings, originally native to Tizira."
```
Comment (Skrem7) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]:
```suggestion
	description = "A savoury glass of tea made from polypore mushroom shavings, originally native to Sangris."
```

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 24]
```diff
@@ -1825,3 +1825,15 @@
 
 #undef PERF_BASE_DAMAGE
 #undef REQUIRED_STRANGE_REAGENT_FOR_REVIVAL
+
+/datum/reagent/medicine/coagulant/seraka_extract
+	name = "Seraka Extract"
+	description = "A deeply coloured oil present in small amounts in Seraka Mushrooms. Acts as an effective blood clotting agent, but has a low overdose threshold."
```
Comment (Skrem7) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]:
```suggestion
	description = "A deeply coloured oil present in small amounts in seraka mushrooms. Acts as an effective blood clotting agent, but has a low overdose threshold."
```

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 25]
```diff
@@ -1825,3 +1825,15 @@
...
+	color = "#00767C"
+	taste_description = "intensely savoury bitterness"
+	glass_name = "glass of seraka extract"
+	glass_desc = "Deeply savoury, bitter, and makes your blood clot up in your veins. A great drink, all things considered."
```
Comment (Skrem7) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]:
```suggestion
	glass_desc = "Deeply savoury and bitter. Slows your blood flow. Dangerous in moderate quantities."
```

**Review by Skrem7 — COMMENTED** [review]
Lore and syntax passover

**Comment by Hopekz** [comment]
> Lore and syntax passover

Oof @ we're just TG lore again I guess

**Review by Bibby0110 — APPROVED** [review]

**Pull request merged**